misc/config_tools/schema/config.xsd

```diff
@@ -99,17 +99,17 @@
     </xs:element>
     <xs:element name="ACPI_PARSE_ENABLED" type="Boolean" default="y">
       <xs:annotation acrn:title="Parse ACPI tables at runtime" acrn:views="advanced">
-        <xs:documentation>Enable ACPI runtime parsing to get DMAR (DMA remapping) configuration data from the APCI tables. Otherwise, use existing, static information from the associated board.xml file.</xs:documentation>
+        <xs:documentation>Enable ACPI runtime parsing to get DMAR (DMA remapping) configuration data from the APCI tables. Otherwise, use existing, static information from the associated board configuration file.</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="L1D_VMENTRY_ENABLED" type="Boolean" default="y">
       <xs:annotation acrn:title="Mitigate L1 terminal fault" acrn:views="advanced">
-        <xs:documentation>Enable L1 cache flush before VM entry,Determine the enabling of mitigation solution for L1 Terminal Fault (L1TF). The L1 data cache will be flushed before each VMENTRY.L1TF is a speculative execution side channel cache timing vulnerability. Each variety of L1TF could potentially allow unauthorized disclosure of information residing in the L1 data cache.</xs:documentation>
+        <xs:documentation>Enable L1 cache flush before VM entry to prevent L1 terminal fault. L1 terminal fault is a hardware vulnerability that allows unauthorized disclosure of information residing in the L1 data cache.</xs:documentation>
       </xs:annotation>
     </xs:element>
     <xs:element name="MCE_ON_PSC_DISABLED" type="Boolean" default="y">
-      <xs:annotation acrn:title="MCE workaround" acrn:views="advanced">
-        <xs:documentation>Enable software workaround for Machine Check Error on Page Size Change (silicon bug in some processor families).</xs:documentation>
+      <xs:annotation acrn:title="Machine Check Error (MCE) workaround" acrn:views="advanced">
+        <xs:documentation>Enable the software workaround for Machine Check Error on Page Size Change (hardware bug in some processor families).</xs:documentation>
```

Comment:
Is that the correct wording (hardware bug)?

Reply:
In my previous project, "silicon" was considered slang.